src/App.js

```diff
@@ -3,22 +3,38 @@ import Home from "./views/Home";
 import CreateArticle from "./views/CreateArticle";
 import { useState, useEffect } from "react";
 
-function App({ prevArticles }) {
+function App() {
   const [articles, setArticles] = useState(
     JSON.parse(localStorage.getItem("articles")) || []
   );
 
+  const [deletedArticles, setDeletedArticles] = useState(
+    JSON.parse(localStorage.getItem("deletedArticles")) || []
```

Comment:
Instead of the `||` I would use the `??`: https://www.joshwcomeau.com/operator-lookup?match=nullish-coalescing